esss_environment.devenv.yml

```diff
@@ -42,7 +42,7 @@ dependencies:
   - attrs>=17
   - colorama
   - invoke>=1.0.0
-  - jinja2==2.10
+  - jinja2>=2.10
```

Comment:
How does this work in py3.6?

Probably some one should fix master so we are able top run test tests to check =P

Reply:
ouch... master is broken for about 4 months... the logs have already been removed...